EIPS/EIP-8.md

```diff
@@ -0,0 +1,148 @@
+### Title
+
+      EIP: 8
+      Title: devp2p Forward Compatibility Requirements for Homestead
+      Author: Felix Lange <felix@ethdev.com>
+      Status: Draft
+      Type: Standards Track
+      Layer: Networking
+      Created: 2015-12-18
+
+### Abstract
+
+This EIP introduces new forward-compatibility requirements for implementations of the
+devp2p Wire Protocol, the RLPx Discovery Protocol and the RLPx TCP Transport Protocol.
+
+### Specification
+
+Implementations of the devp2p Wire Protocol should ignore the version number of hello
+packets. If the remote hello packet contains a version greater than the current local
+version, the implementation should reply with a hello packet containing the local (lower)
+version. Implementations should also ignore any additional list elements at the end of the
+hello packet.
+
+Similarly, implementations of the RLPx Discovery Protocol should ignore the version number
```

Comment:
I will clarify this too. "Ignore" may suggest that the version number should be discarded, which is not the intended meaning. It is simply meant to say that nodes with mismatching version should still be included in the discovery table.

The bit about additional list elements allows for backwards-compatible upgrades in the same manner as for the devp2p hello packet: If any packet is extended to include more information in a later version of the protocol, older nodes will simply ignore these elements.